pkgs/servers/seafile-server/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, python3Packages, autoreconfHook
+, libuuid, sqlite, glib, libevent, libsearpc, openssl, fuse, libarchive, which
+, vala, cmake }:
+
+let
+  libevhtp = import ./libevhtp.nix {
+    inherit stdenv lib fetchFromGitHub cmake libevent;
+  };
+  oniguruma = import ./oniguruma.nix { inherit stdenv lib fetchFromGitHub cmake; };
+in stdenv.mkDerivation rec {
```

Comment:
I'm getting errors when trying some of these programs out:

```
~/nixpkgs ((792b8f64…))> /nix/store/kmhchjnd2ixm8n7bq23p6map8550qwig-seafile-server-8.0.4/bin/seafile-admin --help                                                                                                                                         
  File "/nix/store/kmhchjnd2ixm8n7bq23p6map8550qwig-seafile-server-8.0.4/bin/seafile-admin", line 12
    print 'Python 3 not supported yet. Quit now'
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print('Python 3 not supported yet. Quit now')?
```

And this one says nothing:
```
~/nixpkgs ((792b8f64…))> /nix/store/kmhchjnd2ixm8n7bq23p6map8550qwig-seafile-server-8.0.4/bin/seaf-server --help
```

Reply:
Unfortunately that reflects the state of upstream :unamused: 
`seaf-server` does not seem to respond to --help but requires [several options](https://github.com/haiwen/seafile-server/blob/master/server/seaf-server.c#L36) to run. (see ExecStart of seaf-server for a working example)

As for `seafile-admin`, it's part of the official distribution but also fails on python3 so I guess this is a leftover :shrug:.
It's not mentioned in the documentation either. It looks like something used before [setup-seafile.sh](https://github.com/haiwen/seafile-server/blob/master/scripts/setup-seafile.sh).